packages/tqdm/tqdm.0.1/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+bug-reports:  "https://github.com/LaurentMazare/ocaml-tqdm/issues"
+homepage:     "https://github.com/LaurentMazare/ocaml-tqdm"
+dev-repo:     "git+https://github.com/LaurentMazare/ocaml-tqdm.git"
+maintainer:   "Laurent Mazare <lmazare@gmail.com>"
+authors:      [ "Laurent Mazare" ]
+
+build: [["dune" "build" "-j" jobs "-p" name]]
+
+depends: [
+  "base" {>= "v0.12.0" & < "v0.13"}
+  "dune" {>= "1.3.0"}
+  "ocaml" {>= "4.07"}
+  "ocaml-compiler-libs"
+  "stdio" {< "v0.13"}
+]
+
+synopsis: "OCaml library for progress bars"
+description: """
+A simple OCaml library to display and update progress bars inspired
+by Python tqdm library.
+"""
+
+url {
+  src: "https://github.com/LaurentMazare/ocaml-tqdm/archive/0.1b.tar.gz"
```

Comment:
the tag does not seem to correspond to the version number. Is this correct?

Reply:
Yes this is expected.